esphome/components/sdp3x/sensor.py

```diff
@@ -0,0 +1,32 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import i2c, sensor
+from esphome.const import (
+    CONF_ID,
+    DEVICE_CLASS_PRESSURE,
+    ICON_EMPTY,
+    UNIT_HECTOPASCAL,
+)
+
+DEPENDENCIES = ["i2c"]
+
+sdp3x_ns = cg.esphome_ns.namespace("sdp3x")
+SDP3XComponent = sdp3x_ns.class_("SDP3XComponent", cg.PollingComponent, i2c.I2CDevice)
+
+CONFIG_SCHEMA = (
+    sensor.sensor_schema(UNIT_HECTOPASCAL, ICON_EMPTY, 3, DEVICE_CLASS_PRESSURE)
```

Comment:
Also makes this a statistic that has min/max/average in Home Assistant (needs the import above too)
```suggestion
    sensor.sensor_schema(
        UNIT_HECTOPASCAL, 
        ICON_EMPTY, 
        3, 
        DEVICE_CLASS_PRESSURE, 
        STATE_CLASS_MEASUREMENT,
    )
```